runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -160,6 +175,20 @@ public NettyClientRouter(String host, Integer port) {
         outstandingRequests = new ConcurrentHashMap<>();
         shutdown = true;
 
+        MetricRegistry metrics = CorfuRuntime.getMetrics();
+        String pfx = CorfuRuntime.getMpCR() + host + ":" + port.toString() + ".";
```

Comment:
this prefix should probably be set once and for all in CorfuRuntime.metrics , at initialization time, and not repeat in various places in the code.